spec/features/user_signup_functionality_spec.rb

```diff
@@ -1,11 +1,10 @@
-require "rails_helper"
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.